spotbugs/etc/messages.xml

```diff
@@ -1733,6 +1733,14 @@ factory pattern to create these objects.</p>
       ]]>
     </Details>
   </Detector>
+  <Detector class="edu.umd.cs.findbugs.detect.ForgotToUpdateHashCodeEqualsToStringDetector">
+    <Details>
+		<![CDATA[
+			<p> forgot to update hashCode() or equals() or toString().
```

Comment:
This is not enough detail. Please describe exactly what it is detecting, and how someone should look at addressing it.